packages/shared/src/graphql/posts.ts

```diff
@@ -88,6 +105,7 @@ export interface Post {
   feedMeta?: string;
   downvoted?: boolean;
   flags: PostFlags;
+  userState: PostUserState;
```

Comment:
Hmm because of the anonymous users it could still be optional. But since we enabled mandatory onboarding it could stay as it is..

cc: @rebelchris  ?

Reply:
Mandatory is still only for "new user" existing anonymous users would still be valid.
So indeed should be optional then if we only return it on logged users?